plugins/fields.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: fields
+spec:
+  platforms:
+  - uri: https://github.com/rewanth1997/kubectl-fields/releases/download/v1.2.0-beta/kubectl-fields-Darwin-x86_64.tar.gz
+    sha256: 35072754f24a0bf239c6bb90a53c528b7b88f99435e2a142907599b18169fade
+    bin: kubectl-fields
+    files:
+    - from: "*"
+      to: "."
+    selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+  - uri: https://github.com/rewanth1997/kubectl-fields/releases/download/v1.2.0-beta/kubectl-fields-Linux-x86_64.tar.gz
+    sha256: 730602c8400ad7b712a2eab26d3e1a5feb6593f442f369a3ea246fbb37f6fd3d
+    bin: kubectl-fields
+    files:
+    - from: "*"
+      to: "."
+    selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+  - uri: https://github.com/rewanth1997/kubectl-fields/releases/download/v1.2.0-beta/kubectl-fields-Windows-x86_64.zip
+    sha256: 2ea706370ea3260e9e68943947bbaff295bc4fa1b97cf1492912b01d04d303a2
+    bin: kubectl-fields.exe
+    files:
+    - from: "*"
+      to: "."
+    selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+  version: v1.2.0-beta
+  homepage: https://github.com/rewanth1997/kubectl-fields
+  caveats: |
+    Usage:
```

Comment:
we don't need to copy usage string here.
you'll almost certainly forget to keep it up-to-date.

Reply:
Wow, thanks. That will be really helpful for me. I miss this often.